### ISSUE TITLE
Text wrapping in rename file dialog

### ISSUE THREAD

**Issue by onetrev** [ignism/nucleus-dark-ui#28]
Not sure what can be done here, as I know the large font is part of the UI and what makes it what it is. However in this case I find the text wrapping in this file rename dialog a bit tough to work with. Sorry I don't have any ideas, but thought I should point this out in case you weren't aware. Thanks!

![untitled-1](https://cloud.githubusercontent.com/assets/597601/9974461/43e0e804-5e45-11e5-8003-c3d421559429.jpg)

**Comment by ignism** [comment]
I know it is an issue, but I have trouble reproducing it. It was mentioned before here: https://github.com/ignism/nucleus-dark-ui/issues/4 Are you on OS X?

**Comment by onetrev** [comment]
Sorry, totally my bad. I know better than to not include OS and look for this issue first.

I am on Windows 10. I do also have soft wrap enabled, but disabling that didn't change it.

Looking through the thread in the other issue and then on to Atom, looks like there may have been a temp fix for this posted here: https://github.com/atom/tree-view/issues/473#issuecomment-131388489

Thoughts?

**Comment by ignism** [comment]
Since I can't reproduce it, it's a bit difficult for me to solve this. What you could try is open up the nucleus code, and and the following at the bottom to `atom.less`:

```
atom-text-editor[mini] {
  height: auto;
  max-height: none !important;
}
```

If this works for you, I will implement it in upcoming patch.

**Comment by onetrev** [comment]
Dang. No dice. Bummer you can't reproduce. So your dialog box grows wider or taller as you type a long file name? Looks like you are trying to make it grow taller with this fix. I'll play around with some hacks/fixes too when I can since I have this issue.

**Comment by ignism** [comment]
The overlay stays the same size, but the text input just scrolls horizontally. Without a scrollbar, If you catch my drift. Also, maybe try to override it using `height: auto !important`. It will change the correct height, but if that works I can try to fix the height issue myself.

**Comment by onetrev** [comment]
Gotcha. Ya the `!important` (hack!) on the height works. The dialog continues to grow in height now, so that works as now the text is fully readable as it grows. Hopefully doesn't cause other issues. ;)

**Comment by ignism** [comment]
Ah crap, I was actually hoping it would cause the text to scroll sideways. But it’s a start for now :)
Expect this to be patched tomorrow.

**Comment by ignism** [comment]
I've included the max-height issue in the latest patch, could you check if this works for you? I've also sneaked in `white-space: nowrap;` which I'm hoping helps to not let the editor create a 2 lined dialog. However, if this is stell the case could you tell me if the lines are visible?
(version 0.4.7)

**Comment by onetrev** [comment]
This this is interesting. The dialog is now much wider, so I'd think almost all path+filenames would fit.... However the scroll bar and non expanding height is back.

**Comment by ignism** [comment]
Yes, I thought it was rather small. Never noticed it before, but could not unsee. I patched it again, I think 0.5.1 will work for you.

**Comment by onetrev** [comment]
Perfection! Thank you many times over.
